terraform/modules/spack/gitlab_runner_iam.tf

```diff
@@ -0,0 +1,124 @@
+locals {
+  gitlab_domain = "gitlab${var.deployment_name == "prod" ? "" : ".staging"}.spack.io"
+
+  mirror_roles = {
+    "pr_binary_mirror" = {
+      "role_name_suffix"     = "PRBinaryMirror${var.deployment_name == "prod" ? "" : "-${var.deployment_name}"}",
+      "role_arn_ci_var_name" = "PR_BINARY_MIRROR_ROLE_ARN",
+      "conditions"           = ["project_path:${data.gitlab_project.spack.path_with_namespace}:ref_type:branch:ref:pr*"],
```

Comment:
The rule we use in gitlab is a bit more restrictive about the name of pr branches, but I assume slightly fuller-featured regexes aren't supported here?

Reply:
Correct, unfortunately https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html#Conditions_String is what we're limited to.